themes/article/app/routes/$.tsx

```diff
@@ -133,6 +140,11 @@ export function Article({
   const tree = copyNode(article.mdast);
   const abstract = extractPart(tree, 'abstract');
   const { title, subtitle } = article.frontmatter;
+
+  const thebe = useThebeOptions();
+  const canCompute = !!thebe && (article.frontmatter as any)?.thebe !== false;
+  // TODO in lieu of extended frontmatter or theme options
+  const enable_notebook_toolbar = false;
```

Comment:
@rowanc1 do we have a way of bringing in theme options yet for the web theme? or any thoughts on where to do that?

I am wondering if there is already a mechanism there via the site config + template options?